index.js

```diff
@@ -5,5 +5,5 @@
 module.exports = function(content) {
 	this.cacheable && this.cacheable();
 	this.value = content;
```

Comment:
Good question. It's some webpack 0.9 fix from https://github.com/webpack-contrib/raw-loader/commit/9bbe8b25f979bd039fbd96c40ca55f8142f11353 . I wonder if that line is even needed now.